src/python/ensembl/io/genomio/gff3/process_gff3.py

```diff
@@ -325,6 +333,32 @@ def format_mobile_element(self, feat):
 
         return feat
 
+    def format_gene_segments(self, transcript: SeqFeature) -> SeqFeature:
+        """Returns the equivalent Ensembl biotype feature for gene segment transcript features.
+
+        Supported features: "C_gene_segment" and "V_gene_segment".
+
+        Args:
+            transcript: Gene segment transcript feature.
+
+        """
+        # Change V/C_gene_segment into a its corresponding transcript names
+        if transcript.type in ("C_gene_segment", "V_gene_segment"):
+            standard_name = transcript.qualifiers["standard_name"][0]
+            # Drop "_segment" from the transcript type
+            biotype = transcript.type[:-8]
+            if re.search(r"\b(immunoglobulin|ig)\b", standard_name, flags=re.IGNORECASE):
+                biotype = f'IG_{biotype}'
+            elif re.search(r"\bt[- _]cell\b", standard_name, flags=re.IGNORECASE):
+                biotype = f'TR_{biotype}'
+            else:
+                print(f"Unexpected 'standard_name' content for feature {transcript.id}: {standard_name}")
```

Comment:
We might want to keep this logged instead of just printing, or even exit since we need to check how to handle that